maintainers/maintainer-list.nix

```diff
@@ -11243,6 +11243,15 @@
     githubId = 101514;
     name = "Orivej Desh";
   };
+  ostrolucky = {
```

Comment:
Can you please add your entry at the right please? We try to keep maintainers list sorted alphabetically.

There is also an initiative to keep the list sorted at #218308

Reply:
Yeah  I tried, but apparently sorting changed after rebase. Fixed.